.github/workflows/coverage.yml

```diff
@@ -22,45 +22,3 @@ on:
 env:
   BUILD_TYPE: RelWithDebInfo
   INSTALL_DIR: install
-
-jobs:
-  # Job producing code coverage report
-  # ==================================
-  Code_coverage:
-    # When ubuntu version will be upgrade, you should look if xtensor is now available
-    runs-on: ubuntu-20.04
-
-    steps:
-    - name: "Install libraries"
```

Comment:
Maybe instead of removing it completely let's just disable it by adding: `if: ${{ false }}` (found [here](https://code-maven.com/slides/github-ci/disable-single-github-action-job).